src/main/java/org/bitbuckets/lib/core/LoopDriver.java

```diff
@@ -40,7 +40,7 @@ public void registerLoopPeriodic(int parentId, Runnable loop, int period_ms) {
 
     }
 
-    public void runAlways() {
+    public void runPeriodic() {
```

Comment:
do we have a runAlways somewhere else for logging when disabled? like mechanism sensors?

Reply:
Should clarify that this is always, just wanted to distinguish it from sim

Reply:
👍